{"playernames": [null, "TestBot.py"], "error": "Traceback (most recent call last):\n  File \"/home/cs-students/15mjf2/Desktop/ants/LinuxMacTools/engine.py\", line 93, in run_game\n    sandbox.start(bot_cmd)\n  File \"/home/cs-students/15mjf2/Desktop/ants/LinuxMacTools/sandbox.py\", line 383, in start\n    cwd=working_directory)\n  File \"/usr/lib/python2.7/subprocess.py\", line 679, in __init__\n    errread, errwrite)\n  File \"/usr/lib/python2.7/subprocess.py\", line 1130, in _execute_child\n    executable = args[0]\nIndexError: list index out of range\n"}